testing/osde2e/gpu-addon.sh

```diff
@@ -142,7 +142,7 @@ if [[ ${OCM_ENV} == "int" ]]; then # integration
     echo "====== Skipping RHODS install"
 else
     echo "====== Installing RHODS"
-    run_test "ocm_addon install --ocm_addon_id=managed-odh --ocm_url=${OCM_ENV} --ocm_cluster_id=${CLUSTER_ID} --ocm_addon_params='[{"id":"notification-email","value":"sdayan@redhat.com"}]' --wait_for_ready_state=True"
+    run_test "ocm_addon install --ocm_addon_id=managed-odh --ocm_url=${OCM_ENV} --ocm_cluster_id=${CLUSTER_ID} --ocm_addon_params='[{\"id\":\"notification-email\",\"value\":\"sdayan@redhat.com\"}]' --wait_for_ready_state=True"
```

Comment:
Mmm, I don't like the email address being non-parameterized and public on the internet

Reply:
good point, I did the same "mistake", 
the right place for the email address is in a secret from the vault

Reply:
(but osde2e doesn't use the `ci-operator` and can't use the vault, sorry :# ;-) )

Reply:
This is a dummy email. Ill make it more clear that it is. There is no need for a real email here. 

@kpouget osde2e can get secrets from vault, just a different namespace 😉. Let me know if you want to know how to get that working.

Reply:
> This is a dummy email

you're saying that to confuse the spammers, I know ;-)

(it's your correct address, or I missed something 🙃)

> osde2e can get secrets from vault

oh, great, I didn't know that! 

here is the PR on my side: https://github.com/openshift-psap/ci-artifacts/pull/383
having the email in the secret has the additional benefit that the address will be automatically censored by the CI,
I don't handle it as a real secret (=I pass it as a CLI parameter), but the CI censoring should spot it and replace it by ****

[I'm not using osde2e, I launch the cluster myself]